src/pages/AnalyticsPage/AnalyticsLiquidityChart.tsx

```diff
@@ -112,7 +112,7 @@ const AnalyticsLiquidityChart: React.FC = () => {
       )}
       <Box>
         <Typography style={{ color: palette.text.disabled }} variant='caption'>
-          {moment().format('MMM DD, YYYY')}
+          {dayjs().format('MMM DD, YYYY')}
```

Comment:
Why did you remove moment for dayjs?

Reply:
I removed moment as dayjs has all functionalities that moment has